contents/blog/how-to-pivot.md

```diff
@@ -0,0 +1,214 @@
+---
+date: 2021-01-14
+title: A story about pivots
+rootPage: /blog
+sidebar: Blog
+showTitle: true
+hideAnchor: true
+---
+
+PostHog has pivoted a _lot_.
+
+After 5 pivots in 6 months, we got into YCombinator last year, pivoted again whilst we were there and have now gone from the first commit to $12M raised, thousands of deployments and a team across 10 countries, in well under a year.
```

Comment:
I think the dollar sign needs to be escaped i.e. \$12M

I could get a bot to start doing that